kdc/kerberos5.c

```diff
@@ -919,6 +919,9 @@ pa_enc_ts_validate(astgs_request_t r, const PA_DATA *pa)
 	goto out;
     }
     free_EncryptedData(&enc_data);
+    if (ret) {
```

Comment:
@jsutton24 I am confused.  How could the flow of control arrive at this line with 'ret' non-zero?  If ret is non-zero at line 895 it will enter the block above that either executes a goto try_next_key or goto out at line 919.

Reply:
Sorry, this was a Samba‐only issue, introduced by https://git.samba.org/?p=lorikeet-heimdal.git;a=patch;h=05819c3ae5c6395135dd6611021ac9fe70f3cf53.